docs/UserGuide.md

```diff
@@ -115,17 +114,18 @@ DESCRIPTION OF PARAMETER
 
 
 #### `ROOM_NUMBER`
-DESCRIPTION OF PARAMETER
-* FORMAT AND RESTRICTIONS WITH JUSTIFICATION
-* (if applicable) For best usage, ...
-* (if applicable) Valid examples (if not clear from above)
+Room number for a room
+* Format: `XY-ABC`, where XY can be any pair of digits except 00, and ABC can be any 3 digits.
+    * Valid examples: 01-000, 11-100, 12-345.
+    * Invalid examples: 00-000, 00-100.
+* Room numbers are unique within the SunRez.
+* We disallow floor numbers being 00, as that is not a common practice in Singapore. However, unit numbers can be 000.
```

Comment:
Hmm, should we state that we are targeting Singapore/NUS? Otherwise this may be a different point of contention.

Reply:
Removed the justification and only set up the domain of the parameters.